Makefile

```diff
@@ -12,11 +12,11 @@ include extension-ci-tools/makefiles/duckdb_extension.Makefile
 
 # Custom makefile targets
 data: data_clean
-	python3 scripts/test_data_generator/generate_iceberg.py 0.001 data/iceberg/generated_spec1_0_001 1
-	python3 scripts/test_data_generator/generate_iceberg.py 0.001 data/iceberg/generated_spec2_0_001 2
+	mkdir -p data_generated/data
+	python3 scripts/data_generators/generate_data.py
 
 data_large: data data_clean
-	python3 scripts/test_data_generator/generate_iceberg.py 1 data/iceberg/generated_spec2_1 2
+	python3 scripts/data_generators/generate_data.py
 
 data_clean:
-	rm -rf data/iceberg/generated_*
+	rm -rf data_generated/
```

Comment:
nit: I think it's a bit cleaner to have the generated data live in `data/generated`